addons/website/models/ir_module_module.py

```diff
@@ -422,17 +422,19 @@ def update_theme_images(self):
         for theme in themes:
             terp = self.get_module_info(theme.name)
             images = terp.get('images', [])
+            if all('/' + os.path.join(theme.name, image) in existing_urls for image in images):
+                continue
+            # Images creation order must be the order specified in the manifest
```

Comment:
1) The order it the implicit magic `id desc` order of the attachment records. We cannot swap those around.

2) This might be true for our themes, but we cannot be sure for third party themes.

My guess is that the cover image was used in an older version of the theme selection screen... maybe in some way in the Apps app... and maybe extracted implicitly by the app store... I really have no clue.